src/parser.py

```diff
@@ -0,0 +1,201 @@
+from __future__ import annotations
+import requests as rq
+import json
+from bs4 import BeautifulSoup as bs
+import configuration as cf
+from transformers import pipeline
+
+question_name = "deepset/roberta-base-squad2"
+question_model = pipeline(
+    "question-answering", model=question_name, tokenizer=question_name
+)
+
+
+translation_name = "Helsinki-NLP/opus-mt-ru-en"
+translation_model = pipeline(
+    "translation", model=translation_name, tokenizer=translation_name
+)
```

Comment:
These loadings may require a lot of time and memory. So it is always nice to be sure that loading happens only once for each model.

Reply:
Removed it in one dictionary with function which loads them. Not quite safe, but better than nothing. The problem is i can`t find ```const``` in python...